content/en/examples/service/networking/name-virtual-host-ingress-no-third-host.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: name-virtual-host-ingress-no-third-host
+spec:
+  rules:
+  - host: first.bar.com
+    http:
+      paths:
+      - pathType: Prefix
+        path: "/"
+        backend:
+          service:
+            name: service1
+            port:
+              number: 80
+  - host: second.bar.com
+    http:
+      paths:
+      - pathType: Prefix
+        path: "/"
+        backend:
+          service:
+            name: service2
+            port:
+              number: 80
+  - http:
+      paths:
```

Comment:
Hi @robscott . Thanks for opening this PR.
If you wanted to validate the example configuration files, there is a recently updated test.
The README is found at: `website\content\en\examples\README.md`.

Reply:
I couldn't quite get the test to work, but I think all the examples should be valid. Is there any way this test could run as part of CI?

Reply:
OK. The test has been updated recently. Right now, authors can run the test locally to validate example manifests.
I am guessing that the test may need to be updated.
/cc @tengqm